pxe-server/README.md

```diff
@@ -0,0 +1,38 @@
+<!-- SPDX-FileCopyrightText: (C) 2025 Intel Corporation -->
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+
+# EIM PXE server
+
+This module implements a PXE server as part of Edge Infrastructure Manager.
+The PXE server is used to support legacy PXE boot as another OS provisioning mechanism supported by EIM.
+It runs a local DHCP and TFTP server that exposes PXE boot info and iPXE script that is needed to initiate OS provisioning.
+
+This chart is only intended to be run locally, on-prem.
+It can be deployed as part of the entire EMF orchestrator or as a standalone Helm chart.
+
+## How to run?
+
+The following configuration must be provided to install PXE server. Adjust the configuration parameters to your local on-prem environment.
+
+- `interface` - name of a network interface connected to local L2 network that the DHCP/TFTP server should listen on. All Edge Nodes being provisioned should have L2 connectivity with this network interface. 
+- `bootServerIP` - an IP address assigned to `interface`, from a local subnet.
+- `subnetAddress` - a network address of L2 subnet (without CIDR mask), e.g., 192.168.1.0 or 10.0.0.0. It is used to match incoming DHCP discovery packets.
```

Comment:
is this user configured ? should it be configured in the profile where we enable PXE and Squid so the user can easily set it in one place at orchestrator install ?